docs/custom_conf.py

```diff
@@ -131,13 +131,14 @@
     "tutorials/cloud-init": "../../howto/cloud-init",
     "tutorials/data-science-and-engineering": "../../howto/data-science-and-engineering",
     "tutorials/gpu-cuda": "../../howto/gpu-cuda",
-    "tutorials/vscode": "../../tutorial/develop-with-ubuntu-wsl",
+    "tutorials/vscode": "../../tutorials/develop-with-ubuntu-wsl",
     # the term "guides" was used in the original repo
     "guides/contributing": "../../howto/contributing",
     "guides/install-ubuntu-wsl2": "../../howto/install-ubuntu-wsl2",
     "guides/run-workflows-azure": "../../howto/run-workflows-azure",
     # improved url
-    "tutorial/getting-started": "../../tutorial/getting-started-with-up4w",
+    "tutorials/getting-started": "../../tutorials/getting-started-with-up4w",
+    "tutorials/getting-started": "../../tutorials/getting-started-with-up4w",
```

Comment:
Is one of the duplicates meant to be singular?
```suggestion
    "tutorials/getting-started": "../../tutorials/getting-started-with-up4w",
    "tutorial/getting-started": "../../tutorials/getting-started-with-up4w",
```

Reply:
Good spot @CarlosNihelton . The old tutorial/ links (there aren't many) need to redirect to tutorials/ .

I have fixed in https://github.com/canonical/ubuntu-pro-for-wsl/commit/b43a79fddbdf17113677efdd241de599b0c5ae78